python-data-version-control/src/prepare.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import pandas as pd
+
+folders_to_labels = {"n03445777": "golf ball", "n03888257": "parachute"}
```

Comment:
Let's make this uppercase since we're using it as a constant:

```suggestion
FOLDERS_TO_LABELS = {"n03445777": "golf ball", "n03888257": "parachute"}
```